### DISCUSSION TITLE
error al compilar

### DISCUSSION THREAD

**Discussion by Carlos0276** [community/community#144643] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hola compañeros su ayuda porfa descargue el código de [AstoundingDock](https://github.com/notsonormal/AstoundingDock)
del repositorio de https://github.com/notsonormal/AstoundingDock/tree/master

y me muestra este error al tratar de compilarlo en Visual Studio 2022
soy nuevo en esto y no se que mas hacer gracias:
 
1>------ Se ha iniciado la ejecución del comando Compilar; proyecto: AstoundingDock, configuración: Debug|AnyCPU ------
1>C:\programar\PRUEBAS\AstoundingDock-master\src\AstoundingDock\AstoundingDock.csproj(442,5): error : This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them. For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is ..\packages\Fody.1.29.4\build\portable-net+sl+win+wpa+wp\Fody.targets.
========== Compilar: 0 correcto, 1 erróneo, 3 actualizado, 0 omitido ==========
![2](https://github.com/user-attachments/assets/149e207f-955a-43fd-8b2b-885bc25e12c1)
![1](https://github.com/user-attachments/assets/a0a2038f-98e9-4971-8ca7-97d808994c61)

**Comment by Akash1134** [comment]
Hi @Carlos0276 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
